client/shipping-label/views/purchase/steps/packages/add-item.js

```diff
@@ -32,17 +31,17 @@ const AddItemDialog = ( {
 		return <span className="wcc-move-item-dialog__package-name">{ packageLabels[ pckgId ] }</span>;
 	};
 
-	const renderRadioButton = ( pckgId, itemIdx, item ) => {
+	const renderCheckbox = ( pckgId, itemIdx, item ) => {
 		const itemLabel = packageLabels[ pckgId ]
 			? __( '%(item)s from {{pckg/}}', { args: { item: item.name }, components: { pckg: getPackageNameElement( pckgId ) } } )
 			: item;
 
-		const onChange = () => setAddedItem( pckgId, itemIdx );
+		const onChange = ( event ) => setAddedItem( pckgId, itemIdx, event.target.checked );
 		return (
 			<FormLabel
 				key={ `${ pckgId }-${ itemIdx }` }
 				className="wcc-move-item-dialog__package-option">
-				<FormRadio checked={ pckgId === sourcePackageId && itemIdx === movedItemIndex }
+				<FormCheckbox checked={ ! ! ( ( addedItems || {} )[ pckgId ] || {} )[ itemIdx ] }
```

Comment:
Ah, yes, much better. Or the [modern version](https://twitter.com/paul_irish/status/885975704055062528) 😉